js/models/stock.js

```diff
@@ -1,12 +1,36 @@
 // define your Stock model here
 function Stock(symbol) {
-  this.symbol = symbol;
+  this.symbol = document.getElementsByTagName("Symbol");
   this.numShares = 0;
+  this.name = document.getElementsByTagName("Name");
+  this.price = document.getElementsByTagName("LastPrice");
 
   // your code to search the API and return a Stock object here
 
 }
 
 Stock.prototype = {
-  totalValue: function() { return this.price * this.numShares; }
+  totalValue: function() {
+    return this.price * this.numShares;
+    },
+
+  fetchName: function(callback) {
+    $.ajax({
+      type: "GET",
+      dataType: "json",
+      url: "http://dev.markitondemand.com/Api/v2/Quote/Name"
+    }).done(function(response) {
+      stockModel.loadStock(response);
+      stockModel.render();
```

Comment:
on this line, you are calling the `render()` function which hasn't been defined in your model's constructor function. I think when you create a `StockView()` that function will be contained there. And as a consequence of that, the `render()` function will be called on the stockView and not the model. 

Additionally, stockModel isn't defined anywhere, so on lines 23 you are instantiating a global variable. It will also error out because `loadStock()` is only defined for `Stock()`'s.